cmd/ackdev/cmd/common.go

```diff
@@ -27,8 +27,10 @@ const (
 )
 
 var (
-	homeDirectory     string
-	defaultConfigPath string
+	homeDirectory        string
+	defaultConfigPath    string
+	goPath               = os.Getenv("GOPATH")
```

Comment:
There is a chance `GOPATH` is not set as an env var (that's up to the user, and we all know what users can be like). It appears though Go provides it as part of `go/build` - https://stackoverflow.com/a/32650077/1278784